frontend/components/home.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { pushHistory } from "../actions/routeActions"
+
+const homeHTML = require("./home.html")
+
+const propTypes = {
+  storeState: React.PropTypes.object
+}
+
+class Home extends React.Component {
+  componentDidMount() {
+    this.redirectForLoggedInUser()
+  }
+
+  componentDidUpdate() {
+    this.redirectForLoggedInUser()
+  }
+
+  redirectForLoggedInUser() {
+    if (this.props.storeState.user) {
+      pushHistory("/sites")
+    }
+  }
+
+  render() {
+    return <div dangerouslySetInnerHTML={{ __html: homeHTML }}/>
```

Comment:
Yeah, there's a few things that trip up react. Also note the SVG markup in there.